mkmm

```diff
@@ -53,8 +54,8 @@ __remove() {
 
   if [[ -e "$1" ]]; then
     if findmnt "$1" >/dev/null; then
-      umount "$1"
-      rmdir "$1"
+      umount -v "$1"
+      rmdir -v "$1"
```

Comment:
I'm not a fan of verbose tools. No message = all good. Maybe add a switch (`-i` for info?) or an ENV variable.